inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/AuditIdService.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.service.core;
+
+/**
+ * The service interface for audit id.
+ */
+public interface AuditIdService {
```

Comment:
It is recommended to add the interface related to `AuditId` into the `AuditService`, because the Audit ID is also a part of Audit, and should be put into the same set of interfaces from the perspective of object-oriented and high cohesion.